File: I.py/1178/A.cpp
#include <iostream>
using namespace std;
int main(){
	int n, k, q, m, d, l, i ;
	cin>>n;
	l=0;
	int *a = new int[n];
	
	cin>>q;
	k=q;
	 
	 q=q>>1;
	 
	i=1;
	while (i<n){
	   cin>>d;
	   m+=d;
	   if (d<=q) {
	   	k+=d;
	   	a[l]= i;
	   	l+=1;
	   };
	i+=1;
};
if ( (k<<1) >= m  ){
    cout<<l<<endl;
    
    while (l>0){
       l-=1;
	   cout<<a[l];	
	};
 	
};
	
}

File: I.py/1178/A.py
import sys
r=sys.stdin.read
def cin():
    a=''
    b=r(1)
    while  str.isspace(b):
        b=r(1)
    while not str.isspace(b):
        a+=b
        b=r(1)
    return int(a)

n=cin()

q=cin()

k=q
q=q>>1
a=[1]
i=1
m=0
while i<n:
    i+=1
    l=cin()
    m+=l
    if q>=l:
        print('d',q)
        k+=l
        a.append(i)
if (k<<1)>=m and len(a)>1:
    print(len(a))
    print(*a)
else:
    print(0)


